### ISSUE TITLE
The performance of the model when images are used as input data

### ISSUE THREAD

**Issue by GG-Bonds** [tudelft-iv/view-of-delft-dataset#67]
I want to know how the model trained with the camera sensor performs on the validation set.

I'm using bevdet model training images, but the performance on the validation set is terrible.

Results: 
Entire annotated area: 
Car: 11.423444976076555 
Pedestrian: 0.9228352640879638 
Cyclist: 10.850960762150814 
mAP: 7.732413667438443 
Driving corridor area: 
Car: 28.327499910022556 
Pedestrian: 1.3862024500322372 
Cyclist: 11.777695988222305 
mAP: 13.830466116092367

**Comment by andraspalffy** [comment]
Hi, sorry we do not have the resources to support or debug individual projects. Also, we did not work with camera images in the paper.
